R/registry.R

```diff
@@ -172,8 +172,10 @@ delete_registered_model <- function(name, client = mlflow_client()) {
 #' @param page_token Pagination token to go to the next page based on a
 #'   previous query.
 #' @param client An MLFlow client. Will be auto-generated if omitted.
+#' @param parse Logical indicating whether to return the `registered_models` element
+#' as a list (FALSE) or tibble (TRUE)
 #' @export
-list_registered_models <- function(max_results = 100, page_token = NULL, client = mlflow_client()) {
+list_registered_models <- function(max_results = 100, page_token = NULL, client = mlflow_client(), parse = FALSE) {
```

Comment:
arguably could leave this function as is and just force the user to call `parse()`